src/element.js

```diff
@@ -704,10 +704,14 @@ class XTerminalElementImpl extends HTMLElement {
 		}
 	}
 
-	focusOnTerminal () {
+	focusOnTerminal (double) {
 		if (this.terminal) {
 			this.model.setActive()
 			this.terminal.focus()
+			if (double) {
+				// second focus with send command to pty
+				this.terminal.focus()
+			}
```

Comment:
for some reason calling focus on xterm does not focus the pty unless it is called twice.